getting_started.md

```diff
@@ -52,74 +52,84 @@ using Flux
 ```
 <br>
 
-### Step 2: Create some train data
-For this example, create some random train data `x` and `y` arrays:
+### Step 2: Create the training data
+First, we'll write a function that generates our "true" data. We'll use to use Flux to recover `W_actual` and `b_actual` by looking only at examples of the `actual_model` function.
 
 ```julia
-x = rand(5)
-y = rand(2) 
+W_actual = [1 2 3 4 5;
+            5 4 3 2 1]
+b_actual = [- 1.0; -2.0]
+actual_model(x) = W_actual * x + b_actual
 ```
 <br>
 
+Next, we generate our training data by passing random vectors into the true model. We'll also add Gaussian noise using `randn()` so that it's not *too* easy for Flux to figure out the model.
+
+```julia
+x_train = [ 5 .* rand(5) for _ in 1:10_000 ]
+y_train = [ actual_model(x) + 0.2 .* randn(2) for x in x_train 
```

Comment:
```suggestion
y_train = [ actual_model(x) + 0.2 .* randn(2) for x in x_train]
```
I also think it would help to note what structure `{x,y}_train` actually have. I don't think it's immediately clear that we're dealing with vectors of arrays here, especially since in practice each batch would be a 2D+ array and the outer wrapper "collection" (vector, tuple, dataloader, etc.) is 100% irrelevant.

Reply:
Would it be preferable to switch this to a 2D array and then use `data = zip(eachcol(x_train), eachcol(y_train))` further down? That was one of my initial thoughts.

Reply:
> switch this to a 2D array

Nope, this creates awkwardness when generating examples. I'll just clarify this in prose.